Gems/Atom/RHI/Code/Source/RHI/MultiDeviceDrawPacketBuilder.cpp

```diff
@@ -39,6 +39,32 @@ namespace AZ::RHI
                 m_drawFilterMask };
     }
 
+    MultiDeviceDrawPacketBuilder::MultiDeviceDrawPacketBuilder(const MultiDeviceDrawPacketBuilder& other)
+    {
+        m_deviceMask = other.m_deviceMask;
+
+        m_drawRequests = other.m_drawRequests;
+
+        m_drawPacketInFlight = aznew MultiDeviceDrawPacket;
+        m_drawPacketInFlight->m_drawListMask = other.m_drawPacketInFlight->m_drawListMask;
```

Comment:
This will not work if the `other.m_drawPacketInFlight` is null. Please add a check

Reply:
Good catch :+1: